src/services/doppler-legacy-client.ts

```diff
@@ -877,6 +891,27 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
     return { success: true };
   }
 
+  public async requestSuggestionUpgradeForm(
+    model: RequestUpgradeModel,
+  ): Promise<ReturnUpgradeFormResult> {
+    const response = await this.axios.post('/WebApp/RequestSuggestionUpgradeDemo', {
```

Comment:
Is this a real working endpoint? or we need to work on this separately?

Reply:
Ceci, this endpoint needs to be created in BE. In this case, do I have to talk with Javi/Adriano or just create a task to be developed later?

Reply:
I have created this task : 
https://makingsense.atlassian.net/browse/DW-1046 to create this endpoint 😉